test/common_nn.py

```diff
@@ -2249,6 +2249,52 @@ def fractional_max_pool3d_test(test_case):
         input_size=(),
         desc='scalar',
     ),
+    dict(
+        fullname='Padding1d_circular',
+        constructor=wrap_functional(F.pad_circular, padding=(1,)),
+        input_fn=lambda: torch.arange(6, out=torch.DoubleTensor()).reshape([1, 2, 3]),
+        reference_fn=lambda i, _: padding1d_circular(i),
+        skip_double=TEST_WITH_ROCM,
+        pickle=False,
+    ),
+    dict(
+        fullname='Padding2d_circular',
+        constructor=wrap_functional(F.pad_circular, padding=(1, 1,)),
+        input_fn=lambda: torch.arange(6, out=torch.DoubleTensor()).reshape([1, 3, 2, 1]),
+        reference_fn=lambda i, _: padding2d_circular(i),
+        skip_double=TEST_WITH_ROCM,
+        pickle=False,
+    ),
+    dict(
+        fullname='Padding3d_circular',
+        constructor=wrap_functional(F.pad_circular, padding=(0, 0, 1,)),
+        input_fn=lambda: torch.arange(3, out=torch.DoubleTensor()).reshape([1, 1, 1, 1, 3]),
+        reference_fn=lambda i, _: padding3d_circular(i),
+        skip_double=TEST_WITH_ROCM,
+        pickle=False,
+    ),
+    dict(
+        module_name='Conv1d',
+        constructor_args=(4, 5, 3, 1, 1, 1, 1, True, 'circular'),
+        input_size=(2, 4, 10),
+        cudnn=True,
```

Comment:
It seems that there is an issue on jit side. Although my input is 3 dimensional here but the input to forward(...) method is 5 dimensional.
I found it out as the jit test was suspiciously working for Conv3D and failing for Conv1D and Conv2D.
cc: @ezyang

Reply:
@suo any ideas on why this would happen?

Reply:
It also looks like some other TorchScript tests are failing, but they really should not be.

Reply:
TorchScript tests related to this pull request, @ezyang  ? On phabricator it doesn't show anything failing